### ISSUE TITLE
bug: Calendar month changes when selecting date on Multiple date mode

### ISSUE THREAD

**Issue by jongbonga** [ionic-team/ionic-framework#28602]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#creating-an-issue).
- [X] I agree to follow the [Code of Conduct](https://ionicframework.com/code-of-conduct).
- [X] I have searched for [existing issues](https://github.com/ionic-team/ionic-framework/issues) that already report this problem, without success.

### Ionic Framework Version

v7.x

### Current Behavior

When using the Calendar on multiple date mode, if the initial month is November, selecting a date from a different month navigates the view back to the initial month. 

### Expected Behavior

The calendar month should not navigate automatically when selecting dates from a month that different from the default one.

### Steps to Reproduce

1. Open page with multiple date calendar
2. Navigate to a different month
3. select a date

### Code Reproduction URL

https://ionicframework.com/docs/api/datetime#multiple-date-selection

### Ionic Info

```
Ionic:

   Ionic CLI                     : 6.20.9 (/Users/jonathan/.nvm/versions/node/v16.14.0/lib/node_modules/@ionic/cli)
   Ionic Framework               : not installed
   @angular-devkit/build-angular : 16.2.10
   @angular-devkit/schematics    : 16.2.10
   @angular/cli                  : 16.2.10
   @ionic/angular-toolkit        : 10.0.0

Capacitor:

   Capacitor CLI      : 5.5.1
   @capacitor/android : 5.5.1
   @capacitor/core    : 5.5.1
   @capacitor/ios     : 5.5.1

Utility:

   cordova-res : 0.15.4
   native-run  : 2.0.0

System:

   NodeJS : v16.14.0 (/Users/jonathan/.nvm/versions/node/v16.14.0/bin/node)
   npm    : 8.3.1
   OS     : macOS Unknown
```

**Comment by liamdebeasi** [comment]
Thanks for the report. The problem here is we animate to the new date when the value is changed, but we do not account for multiple selection. This behavior is valuable for when the `value` is set asynchronously on a different month than what it shown.

However, this is confusing when there are multiple dates selected because we do not reasonably know which date to animate to.

**Comment by liamdebeasi** [comment]
Here is a dev build if you are interested in testing the proposed fix: `7.5.8-dev.11701293337.112ba209`

Install Example: `npm install @ionic/angular@7.5.8-dev.11701293337.112ba209`

**Comment by amandaejohnston** [comment]
This has been resolved via https://github.com/ionic-team/ionic-framework/pull/28847 and the fix will be available in a future release of Ionic Framework. Thank you!
